newm-server/src/main/kotlin/io/newm/server/di/DependencyInjectionInstall.kt

```diff
@@ -30,7 +31,8 @@ fun Application.installDependencyInjection() {
             songKoinModule,
             playlistKoinModule,
             cloudinaryKoinModule,
-            awsKoinModule
+            awsKoinModule,
+            idenfyKoinModule
```

Comment:
I usually use trailing commas so that the next time we add to this list, only one line gets added and it's clear what is added.

Reply:
Good point. Added comma for next time.